context/app/static/js/components/cells/CellExpressionHistogram/CellExpressionHistogram.jsx

```diff
@@ -5,8 +5,8 @@ import { useTheme } from '@material-ui/core/styles';
 import Histogram from 'js/shared-styles/charts/Histogram';
 import CellsService from 'js/components/cells/CellsService';
 
-function CellExpressionHistogram({ uuid, geneName }) {
-  const [geneExpressionData, setGeneExpressionData] = useState([]);
+function CellExpressionHistogram({ uuid, cellVariableName }) {
```

Comment:
`cellVariableName` is a good name... Maybe apply it on the python side, too? Your call.